.github/workflows/status_check.yml

```diff
@@ -14,4 +14,9 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.7"
-      - uses: ccdc-opensource/commit-hooks@v2
+      - name: Get the commit message
```

Comment:
I wondered if this was the same message as used for the file size marker?  My expectation was that the same commands would be used for both, but I realise I may be missing something.

Reply:
Yes it's the same message. The action didn't check the commit message (for file size marker, and, for that matter, file size), and I just realised it when doing the JIRA ID.